manifests/database.pp

```diff
@@ -1,17 +1,18 @@
 # Set up the PostgreSQL and Redis databases
 # @api private
 class pulpcore::database {
-  include postgresql::client
-  include postgresql::server
-  postgresql::server::db { $pulpcore::postgresql_db_name:
-    user     => $pulpcore::user,
-    password => postgresql_password($pulpcore::user, $pulpcore::postgresql_db_password),
+  if $pulpcore::postgresql_manage_db {
+    include postgresql::client
+    include postgresql::server
+    postgresql::server::db { $pulpcore::postgresql_db_name:
+      user     => $pulpcore::user,
+      password => postgresql_password($pulpcore::user, $pulpcore::postgresql_db_password),
+    }
   }
 
   pulpcore::admin { 'migrate --noinput':
     unless      => 'python3-django-admin migrate --plan | grep "No planned migration operations"',
     refreshonly => false,
-    require     => Postgresql::Server::Db[$pulpcore::postgresql_db_name],
```

Comment:
@ehelms Is it ok to remove the `require`? I believe we should be able to run migrations still when configured to use an external DB.